### DISCUSSION TITLE
Share multiple albums with one link

### DISCUSSION THREAD

**Discussion by CrysK** [photoprism/photoprism#3519] [Q&A] [answered]
Hello together,
I have the Community Edition, self-hosted.

Would it be possible to share multiple albums with one link?
That there are subfolders with a public link?

An example: for my family I would like to have a link where all events (Christmas, birthdays, trips, ...) are collected. But not in a single folder (album), because this is very messy. But divided, depending on the event. 

Is this possible?

Many thanks and greetings
 Chris

**Comment by lastzero** [comment, marked as answer]
You can manually set the same secret for both albums.
